client/landing/stepper/declarative-flow/internals/steps-repository/launchpad/types.ts

```diff
@@ -3,6 +3,7 @@ export type { Task } from '@automattic/launchpad';
 import { ChecklistStatuses, SiteDetails } from '@automattic/data-stores';
 import { Task } from '@automattic/launchpad';
 import { MinimalRequestCartProduct } from '@automattic/shopping-cart';
+import { ReactNode } from 'react';
```

Comment:
Nit: this should be a type import.
```suggestion
import { type ReactNode } from 'react';
```

Reply:
Fixed https://github.com/Automattic/wp-calypso/pull/86584/commits/ed8db385028cd43eeb3a4d39369dee1b0c9d1c20